ow_lander/action/CameraSetExposure.action

```diff
@@ -0,0 +1,7 @@
+# goal
+bool automatic
```

Comment:
I almost forgot to mention: The unification document calls this `auto`. That's a C++ keyword and causes compile errors, so I called it `automatic` instead.

Reply:
Interesting, but good to know!

Reply:
Wow I never consider C++ keywords could be a problem, but that makes sense.